cmd/containerd/command/service_windows.go

```diff
@@ -49,8 +49,6 @@ var (
 	allocConsole = kernel32.NewProc("AllocConsole")
 	oldStderr    windows.Handle
 	panicFile    *os.File
-
-	service *handler
```

Comment:
@lowenna ptal perhaps you recall if there was a reason for this variable that I'm overlooking